MyWorkout.Web/Controllers/ExercisesController.cs

```diff
@@ -22,6 +22,23 @@ public ExercisesController(IUnitOfWork unitOfWork)
         }
 
         // GET: Exercises
+        //Exersices/CreateFromDayOfWeek/1
+
+        public IActionResult CreateFromDayOfWeek(int? id)
+        {
+            if (id == default(int))
+            {
+                throw new ArgumentNullException(" Day of week id is not set");
```

Comment:
Remove please white space before " Day of week"

Reply:
space has been removed